models/rental.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'book'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  @all = []
+
+  class << self
+    attr_reader :all
+  end
+
+  def initialize(date:, person:, book:)
+    @date = date
+    @person = person
+    @book = book
+    self.class.all << self
```

Comment:
- [ ] Remember the `@rentals` which you initialized in `Person` and I suggested you do same in `Book`? This rental make use of it to  add itself to `@book` and `@person` in order to achieve the `belongs_to` relationship.

See hint below 👇 
```ruby
def initialize(date, book, person)
    .
    .

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
end
```